app/ui-admin/client/rooms/channelSettingsDefault.js

```diff
@@ -15,11 +15,10 @@ Template.channelSettingsDefault.helpers({
 		return Template.instance().editing.get() === field;
 	},
 	roomDefault() {
-		const room = Template.instance().room.get();
-
-		if (room) {
-			return room.default;
-		}
+		return Template.instance().isDefault.get();
+	},
+	isFavorite() {
+		return Template.instance().isFavorite.get();
 	},
 	defaultDescription() {
```

Comment:
Maybe we can change this helper to show if the channel is favorited by default as well

Reply:
Done. Additionally, added a message to the description so users can tell when "Set as default" is true without needing to edit the setting.